src/main/java/io/debezium/connector/spanner/config/BaseSpannerConnectorConfig.java

```diff
@@ -176,6 +177,14 @@ public abstract class BaseSpannerConnectorConfig extends CommonConnectorConfig {
             .withImportance(Importance.LOW)
             .withDescription("Spanner host");
 
+    public static final Field SPANNER_EMULATOR_HOST = Field.create(GCP_SPANNER_EMULATOR_HOST_PROPERTY_NAME)
```

Comment:
Could you please change the property name not to contain the emulator in it, ideally it should be `gcp.spanner.host` and the word emulator should be remove from property names too.
We have a prcedent for it in PubSub sink - https://debezium.io/documentation/reference/2.3/operations/debezium-server.html#address
And only documentation should highlight that this is meant for emulator.

Reply:
`gcp.spanner.host` and `gcp.spanner.emulator.host` are two different properties that need to be treated differently with `builder.setEmulatot(host)` and `builder.setEmulatorHost(emulatorHost)` when building a spanner connection(see this [file](https://github.com/ShuranZhang/debezium-connector-spanner/blob/EmulatorTesting/src/main/java/io/debezium/connector/spanner/db/DatabaseClientFactory.java#L61)). I'm not really sure what other word we can use to distinguish them if not by the configurable property name.
Or could you please suggest another word to use to distinguish between the two parameters other than "emulator" ? Thanks!
.

Reply:
@ShuranZhang ok, in that case could you use `createInternal` method that should make the config option `internal.gcp.spanner.emulator.host` and set it to invisible?

Reply:
@jpechane Thanks for the suggestion! The point for making this Emulator host configurable is not for internal testing purpose. Running our connector against Emulator instance in test env is designed to be a customer facing new feature. Thus I don't feel it's very consistent to use a name starts with `internal.` and set it to invisible. In addition FYI,  we have a precedent for distinguishing between a regular Spanner host and the emulator host by using the word "emulator" for our apache beam connector [here](https://github.com/apache/beam/blob/4600517b4cd5100e9edd10b2ce9e224633fcb469/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerConfig.java#L60
). Let me know if you have other concerns, or you have other suggesting property name for `gcp.spanner.emulator.host`. Thanks!